### DISCUSSION TITLE
Mitmproxy PCAP to Pipeline and Wireshark.

### DISCUSSION THREAD

**Discussion by Maralexbar** [mitmproxy/mitmproxy#6214] [Q&A]
My apologies to the community for this question because my memories it's kind of vague.
Years ago I had found a tutorial where to export the pcap to the pipeline and read it with Wireshark.
Basically everything was done in one command like this:

`mitmproxy --arguments | wireshark --some-argument`

It was several years ago, I don't know if the function was downgraded because currently I can't find anything related.
I only find the SSLLOGKEY method when searching for tutorials.

**Comment by mhils** [comment]
mitmproxy never supported PCAP export – we're on the wrong layer for that. :)
